src/cm1.F

```diff
@@ -754,7 +754,6 @@ program cm1
 !-----------------------------------------------
```

Comment:
Same as `bc.F`. Use my version to replace your existing one but keep some changes you made.